docs/Installation.md

```diff
@@ -507,7 +507,7 @@ sed -i '' '/[[:<:]]engine[[:>:]]/ s/=.*/= MySQL/' /usr/local/etc/zonemaster/back
 Install, configure and start database engine (and Perl bindings):
 
 ```sh
-pkg install mysql57-server p5-DBD-mysql
+pkg install mysql57-server p5-DBD-MariaDB
 sysrc mysql_enable="YES"
 service mysql-server start

```

Comment:
Also here it must be updated.

Reply:
looks like this is a duplicate of https://github.com/zonemaster/zonemaster-backend/pull/849#discussion_r687908555

Reply:
It was not me duplicating.

Reply:
No, this is
```
sysrc mysql_enable="YES"
service mysql-server start
```
It should be mariadb something.

Reply:
It seems not. I tried on FreeBSD 13, and installing MariaDB with `pkg install mariadb105-server` will install `mysql-server` in `/usr/local/etc/rc.d/`.
I can't find anything with `mariadb-`.
This is also the same procedure in this wiki: https://wiki.freebsd.org/MariaDB